assets/src/blocks/amp-story-page/edit.js

```diff
@@ -261,27 +263,55 @@ class EditPage extends Component {
 										label={ __( 'Poster Image (required)', 'amp' ) }
 										help={ __( 'The recommended dimensions for a poster image are: 720p (720w x 1280h)', 'amp' ) }
 									>
+										{
+											! poster &&
+											<Notice status="error" isDismissible={ false } >
+												{ __( 'A poster image must be supplied.', 'amp' ) }
+											</Notice>
+										}
 										<MediaUpload
 											title={ __( 'Select Poster Image', 'amp' ) }
 											onSelect={ ( image ) => setAttributes( { poster: image.url } ) }
 											allowedTypes={ POSTER_ALLOWED_MEDIA_TYPES }
+											modalClass="editor-amp-story-background-video-poster__media-modal"
 											render={ ( { open } ) => (
 												<Button
-													isDefault
+													className={ 'editor-amp-story-page-background ' + ( ! poster ? 'editor-post-featured-image__toggle' : 'editor-post-featured-image__preview' ) }
```

Comment:
Using `classnames()` here might make it more readable.

Something like this:

```js
classnames( 'editor-amp-story-page-background', {
 'editor-post-featured-image__toggle': ! poster,
 'editor-post-featured-image__preview': poster,
} )
```

Reply:
Done in 2b53dd9d.